ext/alias_editor/main.php

```diff
@@ -91,7 +136,17 @@ public function onPageRequest(PageRequestEvent $event)
                 $t->token = $user->get_auth_token();
                 $t->inputs = $_GET;
                 $t->size = $config->get_int('alias_items_per_page', 30);
+				$input = validate_input(["r_oldtag"=>"string,optional", "r_newtag"=>"string,optional"]);
+				$tag_inputs = array();
+				if (isset($_GET["r_oldtag"])) {
+					$tag_inputs["r_oldtag"] = $_GET["r_oldtag"];
+				}
+				if (isset($_GET["r_newtag"])) {
+					$tag_inputs["r_newtag"] = $_GET["r_newtag"];
+				}
+				$t->inputs = array_merge($t->inputs, $input, $tag_inputs);
                 if ($user->can(Permissions::MANAGE_ALIAS_LIST)) {
+					$t->search_url = make_link("alias/list");
```

Comment:
The default target for an HTML form is "whatever page the form is on", which seems correct when we are adding search parameters to the current page - why override that?

Reply:
When I was testing this without that it took me to the home page instead

Reply:
Ooooh do you not have niceurls enabled? Now that you mention it I can see how this would be broken in that case, I think it should be possible to come up with a more general upstream fix that handles all of the MicroCRUD tables at once, lemme try and do that /o/